### ISSUE TITLE
Agree with FluidSynth about a version of SFe to implement

### ISSUE THREAD

**Issue by sylvia-leaf** [SFe-Team-was-taken/SFe#52]

**Comment by stgiga** [comment]
I don't agree on this. They want to port MIDI 2.0 to SF2. Given this, I see no need to limit ourselves.

**Comment by sylvia-leaf** [comment]
If FluidSynth can add MIDI 2.0 support to legacy SF2.0x, then such an implementation would be included in a future version of SFe 4.x. SFe 5, 6, etc. may have MIDI 2.0 from the start once the FluidSynth implementation is widely used.
